src/Features/Core/Portable/CodeFixes/Suppression/AbstractSuppressionCodeFixProvider.PragmaHelpers.cs

```diff
@@ -126,6 +124,24 @@ internal static SyntaxToken GetNewStartTokenWithAddedPragma(SyntaxToken startTok
                 return startToken.WithLeadingTrivia(trivia.InsertRange(index, pragmaTrivia));
             }
 
+            private static bool IsEndOfLineOrHasLeadingEndOfLine(SyntaxTrivia trivia, AbstractSuppressionCodeFixProvider fixer)
```

Comment:
We should probably just move it out of language-specific fixer and use syntax facts service - will do it as part of next fix :)